xml/hwbp_registers.xml

```diff
@@ -965,7 +965,11 @@
             trigger(s) matched.  If the bit is not implemented, it is always 0
             and writing it has no effect.
         </field>
-        <field name="0" bits="XLEN-7:22" access="R" reset="0" />
+        <field name="0" bits="XLEN-7:23" access="R" reset="0" />
+        <field name="clic" bits="22" access="WARL" reset="0">
+            When set, this trigger will fire whenever the attached CLIC
+            signals a trigger.
```

Comment:
It should be stated that the CLIC is optional.  I assume this is hard-wired if there's no CLIC.

Should this be a bit more generic and talk about an "interrupt controller" so that some CLIC competitor can use this (or even just since the existing CLIC will apparently change names)?  Something like:
```suggestion
            This optional bit, when set, causes this trigger to fire whenever an attached
            interrupt controller signals a trigger.
```

Reply:
The generic wording looks good to me.  Also need to change the name of the field from "clic" to something else... "ic" or "intctrl"?